File: pset1/mario/mario.c
#include<cs50.h>
#include<stdio.h>

int main(void)
// get input from user
{
    int input;

    do
    {
        input = get_int("height\n");
    }

    while (input < 1 || input > 8);
    
//establish variables
    int x = 0;
    
    int input1 = input + 1;
    
//major loop
    for (x = 1; x < input1; x++)
    { 
        int spaces = input - x;
//inner loops
        for (int i = 0; i < spaces; i++)
        {
            printf(" ");
        }
    
        for (int j = 0; j < x; j++)
        {
            printf("#");
        }
  
        printf("  ");
    
        for (int j = 0; j < x; j++)
        {
            printf("#");
        }
    
        printf("\n"); 
    }
}
    

File: pset6/mario/more/mario.py
import cs50

# prompt user for height of pyramid
while True:
    height = cs50.get_int("Input height of pyramid (between 1 and 8) ")
    if height > 0 and height < 9:
        break

# print pyramid accordingly

# major loop
for i in range(1, height + 1):
    # spaces on left
    for x in range(height - i):
        print(' ', end='')
        
    # hashes on left    
    for x in range(i):
        print('#', end='')
        
    # spaces in the middle    
    print('  ', end='')
    
    # hashes on right
    for x in range(i):
        print('#', end='')
        
    # new line    
    print('')
